TurkishFilter/sections/specific.txt

```diff
@@ -132,6 +132,9 @@ kalmp3indir.com##a[href^="https://str1.indirdepo.mobi/"]
 mp3indir.rocks###adv
 shunfajinta.com##.widget-content > div[style$=";text-align:center;width:100%;z-index:999999"]
 flowcdn.nl##body > div[style^="z-index:"][style*="position: fixed;"]
+fenokulu.net##a[href^="//www.acilkitap.com/"]
+fenokulu.net##.banner300Soru:upward(div)
```

Comment:
is not too wide target selector?

Reply:
Static images inside poorly made table layout.

Reply:
I don't see `.banner300Soru`

Reply:
Another pages, like 
https://www.fenokulu.net/yeni/Soru/kat/14-Fikralar/

Reply:
```suggestion
fenokulu.net#?#div.SoruDivSatirDis:has(> div[class^="banner"][class$="Soru"]:not([class*=" "]))
```

upward no need "`#?#`"?